lbrynet/core/Wallet.py

```diff
@@ -502,6 +502,10 @@ def claim_new_channel(self, channel_name, amount):
             raise Exception("New channel claim should have no fields other than name")
         log.info("Preparing to make certificate claim for %s", channel_name)
         channel_claim = yield self._claim_certificate(parsed_channel_name.name, amount)
+        if not channel_claim['success']:
+            log.error(channel_claim)
```

Comment:
Is there an error message or something inside channel_claim that could be logged? Logging a dictionary seems strange.